opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -487,6 +487,11 @@ def test_invalid_attribute_values(self):
 
```

Comment:
Would be nice to also add an invalid byte sequence here to assert it doesn't get added

Reply:
Would be good to check that it logs as well. Something like this:
https://github.com/open-telemetry/opentelemetry-python/blob/c074853a7cf9177847017c7656a2ccce461b191a/opentelemetry-auto-instrumentation/tests/test_instrumentor.py#L33-L34

Reply:
I'm not sure how to create a test case with an invalid sequence because an exception is thrown when I try to encode something that would cause problems.